public/caesar-shift/decryption-tests.txt

```diff
@@ -0,0 +1,120 @@
+Hello World,5,czggj rjmgy
+Hello World,6,byffi qilfx
+Hello World,7,axeeh phkew
+Hello World,8,zwddg ogjdv
+Hello World,9,yvccf nficu
+mongodb,-10,wyxqynl
+mongodb,-9,vxwpxmk
+mongodb,-8,uwvowlj
+mongodb,-7,tvunvki
+mongodb,-6,sutmujh
+mongodb,-5,rtsltig
+mongodb,-4,qsrkshf
+mongodb,-3,prqjrge
+mongodb,-2,oqpiqfd
+mongodb,-1,npohpec
+mongodb,0,mongodb
+mongodb,1,lnmfnca
+mongodb,2,kmlembz
+mongodb,3,jlkdlay
+mongodb,4,ikjckzx
+mongodb,5,hjibjyw
+mongodb,6,gihaixv
+mongodb,7,fhgzhwu
+mongodb,8,egfygvt
+mongodb,9,dfexfus
+r a i N b 0 w,-10,b k s x l 0 g
+r a i N b 0 w,-9,a j r w k 0 f
+r a i N b 0 w,-8,z i q v j 0 e
+r a i N b 0 w,-7,y h p u i 0 d
+r a i N b 0 w,-6,x g o t h 0 c
+r a i N b 0 w,-5,w f n s g 0 b
+r a i N b 0 w,-4,v e m r f 0 a
+r a i N b 0 w,-3,u d l q e 0 z
+r a i N b 0 w,-2,t c k p d 0 y
+r a i N b 0 w,-1,s b j o c 0 x
+r a i N b 0 w,0,r a i n b 0 w
+r a i N b 0 w,1,q z h m a 0 v
+r a i N b 0 w,2,p y g l z 0 u
+r a i N b 0 w,3,o x f k y 0 t
+r a i N b 0 w,4,n w e j x 0 s
+r a i N b 0 w,5,m v d i w 0 r
+r a i N b 0 w,6,l u c h v 0 q
+r a i N b 0 w,7,k t b g u 0 p
+r a i N b 0 w,8,j s a f t 0 o
+r a i N b 0 w,9,i r z e s 0 n
+i have a dream,-10,s rkfo k nbokw
+i have a dream,-9,r qjen j manjv
+i have a dream,-8,q pidm i lzmiu
+i have a dream,-7,p ohcl h kylht
+i have a dream,-6,o ngbk g jxkgs
+i have a dream,-5,n mfaj f iwjfr
+i have a dream,-4,m lezi e hvieq
+i have a dream,-3,l kdyh d guhdp
+i have a dream,-2,k jcxg c ftgco
+i have a dream,-1,j ibwf b esfbn
+i have a dream,0,i have a dream
+i have a dream,1,h gzud z cqdzl
+i have a dream,2,g fytc y bpcyk
+i have a dream,3,f exsb x aobxj
+i have a dream,4,e dwra w znawi
+i have a dream,5,d cvqz v ymzvh
+i have a dream,6,c bupy u xlyug
+i have a dream,7,b atox t wkxtf
+i have a dream,8,a zsnw s vjwse
+i have a dream,9,z yrmv r uivrd
+i came/ I saw/ I conquered,-10,s mkwo/ s ckg/ s myxaeobon
+i came/ I saw/ I conquered,-9,r ljvn/ r bjf/ r lxwzdnanm
+i came/ I saw/ I conquered,-8,q kium/ q aie/ q kwvycmzml
+i came/ I saw/ I conquered,-7,p jhtl/ p zhd/ p jvuxblylk
+i came/ I saw/ I conquered,-6,o igsk/ o ygc/ o iutwakxkj
+i came/ I saw/ I conquered,-5,n hfrj/ n xfb/ n htsvzjwji
+i came/ I saw/ I conquered,-4,m geqi/ m wea/ m gsruyivih
+i came/ I saw/ I conquered,-3,l fdph/ l vdz/ l frqtxhuhg
+i came/ I saw/ I conquered,-2,k ecog/ k ucy/ k eqpswgtgf
+i came/ I saw/ I conquered,-1,j dbnf/ j tbx/ j dporvfsfe
+i came/ I saw/ I conquered,0,i came/ i saw/ i conquered
+i came/ I saw/ I conquered,1,h bzld/ h rzv/ h bnmptdqdc
+i came/ I saw/ I conquered,2,g aykc/ g qyu/ g amloscpcb
+i came/ I saw/ I conquered,3,f zxjb/ f pxt/ f zlknrboba
+i came/ I saw/ I conquered,4,e ywia/ e ows/ e ykjmqanaz
+i came/ I saw/ I conquered,5,d xvhz/ d nvr/ d xjilpzmzy
+i came/ I saw/ I conquered,6,c wugy/ c muq/ c wihkoylyx
+i came/ I saw/ I conquered,7,b vtfx/ b ltp/ b vhgjnxkxw
+i came/ I saw/ I conquered,8,a usew/ a kso/ a ugfimwjwv
+i came/ I saw/ I conquered,9,z trdv/ z jrn/ z tfehlvivu
```

Comment:
Add some unicode and see how it does. A couple possibilities:

café,4,gejé
😃 guy,1,😃 hvz

Other than that, LGTM for a test suite.